pkg/ruler/manager_test.go

```diff
@@ -29,7 +31,10 @@ func TestSyncRuleGroups(t *testing.T) {
 	m, err := NewDefaultMultiTenantManager(Config{RulePath: dir}, factory, nil, log.NewNopLogger(), nil)
 	require.NoError(t, err)
 
-	const user = "testUser"
+	const (
+		user  = "testUser"
+		user1 = "testUser1"
```

Comment:
[nit] I suggest to apply the [boy scout rule](https://deviq.com/principles/boy-scout-rule) here, and cleanup a bit the code for readability:

```suggestion
		user1  = "testUser1"
		user2 = "testUser2"
```